main.js

```diff
@@ -1,8 +1,11 @@
 import { dzis } from './data.js';
 import ImageRec from './ImageRec.js';
 import { prepareIntro } from './intro.js';
+import { highlights, getHighlightBounds, stopAnimation } from './highlights.js';
+import { isTouchDevice, hasMouse } from './util.js';
 
 // Configuration for arranging images
+const showHighlights = false;
```

Comment:
Set this to true to be able to see where your highlights are. This will help with the trial and error of figuring out the right numbers!